source/DistanceAndDirection/DistanceAndDirectionLibrary/Views/RangeView.xaml

```diff
@@ -135,9 +135,12 @@
                 </ComboBox>
                 <TextBlock x:Name="NumberofRadialsLabel"
                            Margin="3,3,0,0"
-                           Text="{x:Static prop:Resources.LabelRangeRingsNumberofRadials}" />
+                           Text="{x:Static prop:Resources.LabelRangeRingsNumberofRadials}" 
```

Comment:
A comment about why hiding (and perhaps even link to issue: https://github.com/Esri/distance-direction-addin-dotnet/issues/181 ) might also be good here